### PR TITLE
Set TERM and PS1 env in Dockerfile

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -26,6 +26,8 @@ ENV GOPATH="/go-workspace"
 ENV PATH="${GOPATH}/bin:/usr/local/go/bin:${PATH}"
 ENV GIT_COMMITTER_NAME="Kubernetes Publisher"
 ENV GIT_COMMITTER_EMAIL="k8s-publishing-bot@users.noreply.github.com"
+ENV TERM=xterm
+ENV PS1='\h:\w\$'
```
[the remark this answers is not in the document]
Comment (sttts) [reply Dockerfile]: Does docker really create layers for that?????

**Comment by sttts** [comment]
> SHELL=/bin/bash maybe :-))

done

File: Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (sttts) [reply Dockerfile]: It doesn't look like it does. Still seeing 8 layers.

[the remark this answers is not in the document]
Comment (mfojtik) [reply Dockerfile]: it used to, maybe they changed it

**Pull request merged**